### DISCUSSION TITLE
Kindle App does not run anymore

### DISCUSSION THREAD

**Discussion by foojak** [MustardChef/WSABuilds#480] [Q&A] [answered]
Hello everyone, I hope you can help me with an issue I'm experiencing. I'm currently using WSA on Windows 11, specifically version WSA_2311.40000.5.0_x64_Release-Nightly-GApps-13.0-NoAmazon, which includes Google Play but not the Amazon Store. Everything was working perfectly until Sunday morning when the Amazon Kindle app suddenly stopped functioning: the app starts with the reading child on blue background, but that's it, nothings else happens, or the app just crashes after some moments. All my other apps are still running without any problems. Here’s what I’ve tried so far:

    Cleared the cache of the Kindle app.
    Cleared all data from the Kindle app.
    Uninstalled and reinstalled the Kindle app.
    Restored a complete Windows backup.

Unfortunately, none of these steps have resolved the issue. My next plan is to reinstall WSA with the current version that includes the Amazon Store, so I can install the Kindle app directly from there instead of Google Play. However, I'm unsure if the Kindle app is still available in the Amazon Store as it was a few weeks ago. I have little hope that this will solve the problem, as the Kindle app is one of the main reasons I started using WSA. I’ve been quite dissatisfied with the Kindle for PC software. Any advice or suggestions would be greatly appreciated! Thank you!

**Comment by MustardChef** [comment, marked as answer]
@jagonda Update to the latest Non-LTS and LTS Builds.

**Reply by foojak** [reply]
Thanks a lot, it works!
